### DISCUSSION TITLE
Winui3 File / Folder Requester

### DISCUSSION THREAD

**Discussion by ward-horsfall** [microsoft/microsoft-ui-xaml#7804] [Q&A]
Hi,

In WinUI3 how do the following:

1. A File requestor?
2. A Folder browser?

Thanks
Ward

**Comment by castorix** [comment]
FileOpenPicker 
FolderPicker

**Reply by shelllet** [reply]
> FileOpenPicker FolderPicker

These two class don't support unpackaged application  elevated administrator

**Reply by castorix** [reply]
> > FileOpenPicker FolderPicker
> 
> These two class don't support unpackaged application elevated administrator

Yes, otherwise [IFileDialog](https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nn-shobjidl_core-ifiledialog) works fine
(tested on Windows 10 21H1 as Admin)
